### ISSUE TITLE
RangeError after adding additional tiles

### ISSUE THREAD

**Issue by shtaffa** [rivafarabi/deckboard#341]
I recently purchased Deckboard Pro.  After doing so I made my first 5x3 board.  I had no problems adding the first 7 items to that board.  Once I add an eighth then I start having issues.  First, if the app is open on my tablet then it immediately crashes.  When I open it back up and go to the 5x3 board then I get a screen full of errors stating:

RangeError (index): Invalid value: Valid value range is empty: *

The * in the error changes.  It's always a number.  Depending on what the 8th item I've added was there are a couple of things that could happen.  Either the application will completely crash after sitting on the error screen for about 10 seconds or will will refresh and pull in the button.  I'm trying to create a board full of sound effects for my stream and unfortunately that makes the app completely crash.

My config:
Desktop: Windows 10 Pro x64
Deckboard desktop version: 1.9.1
Tablet: Sony Experia SGP561, Android 5.1.1
Deckboard tablet version: 1.9.2
Connectivity: USB

**Comment by rivafarabi** [comment]
Thanks for the info. I will look into it. btw, is the the bug only appear on 5x3 grid layout?

**Comment by shtaffa** [comment]
No.  I just switched back to the 4x3 layout and I had the same issue.

**Comment by SPAWNgr** [comment]
i have the same issue isn't the layout the real problem is the background if you change the background to a simple color works fine :)

**Comment by shtaffa** [comment]
I'm not sure what you mean by a simple color.  I did have the sounds page set with a unique background color.  I changed it to the default background and that made no difference.

**Comment by rivafarabi** [comment]
Can you send the database.db file for me to check? It is located in C:\Users\{USER}\deckboard

**Comment by shtaffa** [comment]
I'm unable to attach the file to a message here.  When I zip it the file is still 32.4 MB.  Github only allows 10 MB file uploads.
